### ISSUE TITLE
need .NET Core SDK for ARM to build ARM image

### ISSUE THREAD

**Issue by adashen** [dotnet/dotnet-docker#455]
## Steps to reproduce the issue
1. Create a .net core project
2. The target platform is arm32v7
3. And user want to build the binary in container
4. There is no base image for .net core sdk on arm

## Expected behavior
Provide the base image for .net core sdk on arm

## Actual behavior
There is no base image for .net core sdk on arm. 

## Additional information (e.g. issue happens only occasionally)
Although the workaround to build binary in container for arm is build the image on a linux machine. But there are some cases the workaround will fail
1. When user want to add more packages during run time. Thus the image cannot be built on linux for arm. Since apt-get is not cross platform.

**Comment by MichaelSimons** [comment]
This is a duplicate of https://github.com/dotnet/dotnet-docker/issues/440.

@adashen - Can you elaborate on 

> When user want to add more packages during run time. Thus the image cannot be built on linux for arm. Since apt-get is not cross platform.

Would a multi-stage build help you out?

**Comment by adashen** [comment]
@MichaelSimons , our docker file looks like the following. It is multi-stage build. First we build the the binary in container, second, we copy the binary and run it. If we would like to add "apt-get XXXXX" stuff in the second step. The second step cannot built on linux container. But for step 1 it can only do on linux, since sdk does not support arm.


//////////////////////////////////////Docker file//////////////////////////////////////////
FROM microsoft/dotnet:2.0-sdk AS build-env
WORKDIR /app

COPY *.csproj ./
RUN dotnet restore

COPY . ./
RUN dotnet publish -c Release -o out

FROM microsoft/dotnet:2.0-runtime
WORKDIR /app
COPY --from=build-env /app/out ./
ENTRYPOINT ["dotnet", "test.dll"]

**Comment by MichaelSimons** [comment]
@adashen - To unblock yourself, you could build your own ARM SDK image.  Start with [this](https://github.com/dotnet/dotnet-docker/pull/442).  Alternatively, if you have a win10 or macOS machine setup with Docker, I believe you will be able to build the image you want.  They should support your scenario.
